sumOfNumbers.js

```diff
@@ -0,0 +1,4 @@
+const findSum = (array, i) => {
+  if (i < 0) return 0;
```

Comment:
The function you've written is almost correct, but it will not give the expected output in its current form. The main issue is that it lacks a proper stopping condition, which is essential for any recursive function. Without this, the function will continue to call itself indefinitely, leading to an infinite loop and eventually causing a "Maximum call stack size exceeded" error.